### PR TITLE
Add vertical kern to \vdots

### LINKED ISSUE #620: Add kerning to \vdots
TL;DR: we need `\kern 6.0` on top of our `\vdots`.

Running `pdflatex` on the following code:
```
\documentclass{article}
\usepackage{amsmath}
\begin{document}
\tracingonline=1
\showboxbreadth=\maxdimen
\showboxdepth=\maxdimen
$\vdots \showlists$
\end{document}
```
produced the following output
```
\mathord
.\vbox(15.05554+0.0)x2.77779
..\kern 6.0
..\hbox(1.05554+0.0)x2.77779
...\OT1/cmr/m/n/10 .
..\glue(\baselineskip) 2.94446
..\hbox(1.05554+0.0)x2.77779
...\OT1/cmr/m/n/10 .
..\glue(\baselineskip) 2.94446
..\hbox(1.05554+0.0)x2.77779
...\OT1/cmr/m/n/10 .
```

### PR DESCRIPTION
**Pull request by ronkok**
Fixes #620.

vdots

### CODE REVIEW

File: src/macros.js [hunk 1]
```diff
@@ -238,6 +238,16 @@ defineMacro("\u27C2", "\\perp");
...
 defineMacro("\u220C", "\\notni");
 
+//////////////////////////////////////////////////////////////////////
+// LaTeX_2ε
+
+// \vdots{\vbox{\baselineskip4\p@  \lineskiplimit\z@
+// \kern6\p@\hbox{.}\hbox{.}\hbox{.}}}
+// We'll call \varvdots, which gets a glyph from symbols.js.
+// The zero-width rule gets us an equivalent to the vertical 6pt kern.
+defineMacro("\\vdots", "\\mathord{\\varvdots\\rule{0pt}{15pt}}");
```
[the remark this answers is not in the document]
Comment (ronkok) [reply src/macros.js]: `\kern` provides horizontal space. I'm not aware of any way to make it provide a vertical spacing function. Am I missing something?

File: src/symbols.js [hunk 2]
```diff
@@ -655,7 +655,7 @@ defineSymbol(text, main, inner, "\u2026", "\\ldots", true);
 defineSymbol(math, main, inner, "\u2026", "\\ldots", true);
 defineSymbol(math, main, inner, "\u22ef", "\\@cdots", true);
 defineSymbol(math, main, inner, "\u22f1", "\\ddots", true);
-defineSymbol(math, main, textord, "\u22ee", "\\vdots", true);
+defineSymbol(math, main, textord, "\u22ee", "\\varvdots"); // \vdots is a macro
```
[the remark this answers is not in the document]
Comment (ronkok) [reply src/symbols.js]: `\u22ee`, as a macro, will expand to `\vdots`. Then the `\vdots` macro will call the symbol now designated as `\varvdots`.

I'm open to another alias for `\varvdots`, by the way. I just needed a way give the macro and the symbol different names.

[the remark this answers is not in the document]
Comment (kevinbarabash) [reply src/symbols.js]: My mistake.  That makes sense.

File: src/macros.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevinbarabash) [reply src/macros.js]: I ran `\vbox{\hbox{a}\kern6\hbox{b}}` on quicklatex.com and it resulted in a vertical space between the `a` and the `b`.  It looks like our `\kern` doesn't support this ability yet.

**Comment by ronkok** [comment]
I like the easy ones.

**Pull request merged**